lib/utils.js

```diff
@@ -565,7 +566,11 @@ exports.lookupFiles = function lookupFiles(filepath, extensions, recursive) {
     }
     if (!extensions) {
       throw createMissingArgumentError(
-        'Argument "extensions" required when argument "filepath" is a directory',
+        util.format(
+          'Argument %s required when argument %s is a directory',
+          exports.sQuote('extensions'),
+          exports.sQuote('filepath')
+        ),
```

Comment:
Does this work, single quotes within single quotes?

Reply:
Mais bien sûr!